labs/sysdev-kernel-cross-compiling/sysdev-kernel-cross-compiling.tex

```diff
@@ -87,8 +98,7 @@ \section{Linux kernel configuration}
 kernel.}{}
 
 \ifdefstring{\labboard}{beaglebone}
-{In our case look for a configuration for boards in the OMAP2 and
-later family which the AM335x found in the BeagleBone belongs to.}{}
+{The standard configuration for this kernel is actually \code{omap2plus_defconfig}.}{}
```

Comment:
Same, we don't want to give the direct answer. It's intentional that we only give hints to find the solution.

Reply:
Okay, but it is specified for `stm32mp1`, `beagleplay` and `qemu`.